app/networkapi/people/models.py

```diff
@@ -97,6 +97,12 @@ class Person(SortableMixin):
         null=True,
         blank=True,
     )
+    interview_url = models.URLField(
+        max_length=500,
```

Comment:
You know, I thought about this as I was typing it but was just looking at the fields above it for reference. Thought maybe it was a weird python/db thing.